src/components/common/footer.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link';
+import Icon from '@/components/common/Icon';
+
+export default function Footer() {
+  return (
+    <footer className='fixed bottom-0 grid h-116 w-full grid-cols-2 items-center border-1 border-gray-100 bg-white py-10 md:flex md:justify-between md:px-200'>
```

Comment:
<img width="1218" height="168" alt="image" src="https://github.com/user-attachments/assets/a9a3d1b8-69e9-452c-a75e-dce5b1ae13c6" />

```suggestion
    <footer className='fixed bottom-0 grid h-116 w-full grid-cols-2 items-center border-1 border-gray-100 bg-white py-10 md:flex md:justify-between md:px-40 lg:px-200'>
```
tablet에서도 px가 200으로 설정되어 있어서 조금 이상하게 보이네요!
figma에서는 태블릿의 px가 40으로 설정되어 있습니다! 참고해서 수정 부탁드립니다~ 😊